easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.5-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,35 @@
+# initially generated with eb ABINIT-7.10.4-intel-2015a.eb --try-software-version=7.10.5
+# --try-toolchain=intel,2016.02-GCC-4.9, then added intel mpi and fft support
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = "7.10.5"
+
+homepage = 'http://www.abinit.org/'
+description = """ABINIT is a package whose main program allows one to find the total energy,
+ charge density and  electronic structure of systems made of electrons and nuclei (molecules
+ and periodic solids) within Density Functional  Theory (DFT), using pseudopotentials and a
+ planewave or wavelet basis."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://ftp.abinit.org/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = []
+
+# ensure mpi and intel toolchain
+configopts = '--with-fc-vendor=intel --with-linalg-flavor=mkl --enable-mpi ' \
+             '--with-fft-flavor=fftw3-mkl ' \
+             '--with-fft-libs="-lmkl_intel_lp64 -lmkl_sequential -lmkl_core"'
```

Comment:
any particular reason you're not using `$LIBFFT` here, that is defined by the toolchain mechanism?

```
export LIBFFT="-Wl,-Bstatic -Wl,--start-group -lfftw3xc_intel -lfftw3x_cdft_lp64 -lmkl_cdft_core -lmkl_blacs_intelmpi_lp64 -lmkl_intel_lp64 -lmkl_sequential -lmkl_core -Wl,--end-group -Wl,-Bdynamic"
```

(obtained with `eb --from-pr 2894 -x` in an environment where `intel/2016.02-GCC-4.9` is readily available)